pycircleci/api.py

```diff
@@ -747,6 +748,48 @@ def delete_envvar(self, username, project, name, vcs_type=GITHUB):
         resp = self._request(DELETE, endpoint)
         return resp
 
+    def add_context(self, name, username, vcs_type=GITHUB):
+        """Get contexts for an organization
+
+        :param name:  Context name to add.
+        :param username: Org or user name.
+        :param vcs_type: VCS type (github, bitbucket). Defaults to ``github``.
+
+        Endpoint:
+            POST ``/v2/context``
+        """
+
+        slug = self.owner_slug(username, vcs_type)
+        data = {
+            "name": name,
+            "owner": {"id": slug, "type": "organization"}
```

Comment:
can use ORGANIZATION ?

Reply:
I'll allow `owner_type` here too, and rework it to match the other one in terms of supporting both organization and account type.

I can't test the self-hosted Circle thing, but I guess someone will tell us if it's wrong.

It seems like the id can take either slug or uuid here, which is not totally consistent with how the list contexts method works 🤷.